### DISCUSSION TITLE
Adventure map, Xanadu, display the primary skills icons (without the white selection frame) and "+1" under each icon when the hero gets the bonus [Done]

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#4877] [Ideas]
I suggest to display the four primary skills small icons (as in Arena window, without the white selection frame) and "+1" (without the skills name) under each icon when the hero gets the bonus.

In fheroes2:

![Xanadu icons 001](https://user-images.githubusercontent.com/43583994/147853464-0e321ea4-98f8-4651-9d5c-f04b1536d214.png)

![Xanadu icons 002](https://user-images.githubusercontent.com/43583994/147853470-83f0f3f6-b64f-48a9-80c4-ef63fdb639c2.png)

**Comment by zenseii** [comment]
The biggest problem I have with using the same setup for the Xanadu as the Arena is that it could confuse players to think that you have to pick one of the four from the Xanadu. It's still better than nothing though and I'd have to try it and see to know if it's confusing.

**Reply by Branikolog** [reply]
Hi, @zenseii !
I think it could be a bit confusing, especially, comparing to arena.
As a solution these icons could be united into one block or frame, so they would look as a single element.

**Reply by Alucard648** [reply]
And add +! at bottom of each frame.

**Reply by LeHerosInconnu** [reply]
Hello @zenseii, @Branikolog and @Alucard648,

I think it would be okay without the white selection frame and with the "+1" added.
The first time the player will read the text and he should understand.
Then, even if he clicks on the graphics it won't change anything, the hero will have +1 in the four primary skills.
After that, the player will identify all "+1" visually and will not need to read the text anymore.

**Comment by LeHerosInconnu** [comment]
This has been implemented.
I close the discussion.
